widget/ui/src/components/Alert/Alert.helpers.ts

```diff
@@ -15,3 +15,12 @@ export const getColor = (type: Type, variant: 'alarm' | 'regular') => {
       return undefined;
   }
 };
+
+export const mapVariantToSize = (variant: 'alarm' | 'regular') => {
```

Comment:
Please use `PropTypes["variant"]` for the type.

Reply:
Done